src/NuGet.Services.V3/Registration/Models/RegistrationPackageDependency.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Protocol.Catalog;
+
+namespace NuGet.Protocol.Registration
+{
+    public class RegistrationPackageDependency : PackageDependency
+    {
+        [JsonProperty("registration", Order = 1)]
```

Comment:
Just curious, what's the `Order = 1` for? Does it make the `Registration` property the first serialized property?

Reply:
It makes that property last, which is what the existing order on nuget.org has.